theano/tensor/tests/test_opt.py

```diff
@@ -2502,8 +2518,9 @@ def test_none_slice(self):
             sub_x = x[slice1][slice2]
             f = theano.function([x] + input_vars, sub_x, mode=mode_opt)
 
-            self._verify_stack_trace(f)
-
+            # Check stacktrace was copied over correctly after opt was applied
```

Comment:
Please leave a comment saying that for some cases, the optimization may remove all Subtensors, which is why you pass "bug_print='ignore'".